core/src/apps/cardano/layout.py

```diff
@@ -62,10 +64,22 @@ def format_coin_amount(amount: int) -> str:
     return "%s %s" % (format_amount(amount, 6), "ADA")
 
 
-async def confirm_sending(ctx: wire.Context, amount: int, to: str) -> None:
+def is_printable_ascii_bytestring(bytestr: bytes) -> bool:
+    return all((32 < b < 127) for b in bytestr)
```

Comment:
changed this from `32 <= b` to `32 < b` as I realized that would allow spaces which are not really printable/visible

Reply:
so a token name like `"Some Token"` is not allowed?

Reply:
ping @refi93

Reply:
it is but it will fall back to being shown in hex to avoid confusion leading to possible exploits as it may get confused e.g. with `Some  Token`